[2025-06-14T15:06:19.727+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_automation.stream_data_from_api manual__2025-06-14T15:06:17.349513+00:00 [queued]>
[2025-06-14T15:06:19.732+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_automation.stream_data_from_api manual__2025-06-14T15:06:17.349513+00:00 [queued]>
[2025-06-14T15:06:19.733+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-06-14T15:06:19.750+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): stream_data_from_api> on 2025-06-14 15:06:17.349513+00:00
[2025-06-14T15:06:19.757+0000] {standard_task_runner.py:57} INFO - Started process 225 to run task
[2025-06-14T15:06:19.761+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'user_automation', 'stream_data_from_api', 'manual__2025-06-14T15:06:17.349513+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/kafka_stream.py', '--cfg-path', '/tmp/tmpmmt0w890']
[2025-06-14T15:06:19.763+0000] {standard_task_runner.py:85} INFO - Job 7: Subtask stream_data_from_api
[2025-06-14T15:06:19.825+0000] {task_command.py:410} INFO - Running <TaskInstance: user_automation.stream_data_from_api manual__2025-06-14T15:06:17.349513+00:00 [running]> on host aa93a304a005
[2025-06-14T15:06:20.269+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='user_automation' AIRFLOW_CTX_TASK_ID='stream_data_from_api' AIRFLOW_CTX_EXECUTION_DATE='2025-06-14T15:06:17.349513+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-14T15:06:17.349513+00:00'
[2025-06-14T15:06:20.616+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=broker:29092 <connecting> [IPv4 ('10.89.0.17', 29092)]>: connecting to broker:29092 [('10.89.0.17', 29092) IPv4]
[2025-06-14T15:06:20.652+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=broker:29092 <checking_api_versions_recv> [IPv4 ('10.89.0.17', 29092)]>: Broker version identified as 2.6
[2025-06-14T15:06:20.654+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=broker:29092 <connected> [IPv4 ('10.89.0.17', 29092)]>: Connection complete.
[2025-06-14T15:08:27.655+0000] {kafka_stream.py:56} ERROR - an error occurred: HTTPSConnectionPool(host='randomuser.me', port=443): Max retries exceeded with url: /api/ (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1129)')))
[2025-06-14T15:08:28.416+0000] {conn.py:945} ERROR - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=broker:29092 <connected> [IPv4 ('10.89.0.17', 29092)]>: Closing connection. KafkaConnectionError: Socket EVENT_READ without in-flight-requests
[2025-06-14T15:08:28.568+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-06-14T15:08:29.888+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=user_automation, task_id=stream_data_from_api, execution_date=20250614T150617, start_date=20250614T150619, end_date=20250614T150829
[2025-06-14T15:08:30.415+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-06-14T15:08:30.546+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
